### DISCUSSION TITLE
What means the Polaris Catalog announcement?

### DISCUSSION THREAD

**Discussion by alvarosantossyngenta** [projectnessie/nessie#9229] [General]
As a current user of Nessie in my private projects, It is important to understand the implications of the Polaris Catalog [announcement](https://www.snowflake.com/blog/polaris-catalog-open-source/) ("The team at Dremio is excited to help bring the various functions and capabilities of Nessie into the Polaris  project"). Can you elaborate it from Nessie / Dremio perspective?

**Comment by snazy** [comment]
Hey @alvarosantossyngenta,

we have posted a response in the [Nessie blog](https://projectnessie.org/blog/)

**Reply by alvarosantossyngenta** [reply]
Thanks for the clarification.
